examples/benchmark/profet_benchmark.py

```diff
@@ -0,0 +1,167 @@
+import logging
+from dataclasses import dataclass
+from logging import getLogger as get_logger
+from pathlib import Path
+from typing import List, Type
+
+import numpy as np
+import pandas as pd
+import plotly
+from orion.benchmark.assessment import AverageRank, AverageResult
+from orion.benchmark.assessment.warm_start_task_correlation import (
+    warm_start_task_correlation_figure,
+)
+from orion.benchmark.benchmark_client import get_or_create_benchmark
+from orion.benchmark.task import BaseTask, CarromTable, EggHolder, RosenBrock
+from orion.benchmark.task.profet import FcNetTask, ForresterTask, SvmTask, XgBoostTask
+from orion.benchmark.task.profet.profet_task import MetaModelTrainingConfig, ProfetTask
+from orion.benchmark.task.quadratics import QuadraticsTask
+from orion.benchmark.task.task import Task
+from orion.benchmark.warm_start_benchmark import WarmStartTaskCorrelationBenchmark
+from simple_parsing.helpers import choice, list_field
+
+logger = get_logger("orion.benchmark.task")
+logger.setLevel(logging.INFO)
+
+from orion.benchmark.task.profet import FcNetTask, ForresterTask, SvmTask, XgBoostTask
+
+# BUG: #629 (https://github.com/Epistimio/orion/issues/629)
+from orion.core.worker.primary_algo import PrimaryAlgo
+from orion.testing.space import build_space
+_ = PrimaryAlgo(space=build_space(), algorithm_config="random")
+
+
+@dataclass
+class ProfetExperimentConfig:
+    """ Configuration option for the demo of the Profet tasks. """
+
+    # The type of Profet task to create.
+    task_type: Type[Task] = choice(
```

Comment:
This is actually assigned the task constructor right? Maybe task_constructor would be less confusing unless I am wrong.

Reply:
You're right! However I call it "task" so that you can use `--task carrom_table`, etc